extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -394,7 +398,7 @@ class AmpAd extends AMP.BaseElement {
         this.toggleFallback(true);
       }
       // Remove the iframe only if it is not the master.
-      if (this.iframe_.name.indexOf('_master') == -1) {
+      if (this.iframe_ && this.iframe_.name.indexOf('_master') == -1) {
```

Comment:
What should be the desired state here? If the user navigates back to this tab, they may see the fallback element now whereas usually we would try loading another ad.